src/components/BrandConfigProvider/theme/base/light.ts

```diff
@@ -59,3 +59,15 @@ export const colors = {
 };
 
 export type Colors = typeof colors;
+
+export const spacing = {
+  micro: 2,
+  tiny: 4,
+  extraSmall: 8,
+  small: 12,
+  medium: 16,
+  large: 24,
+  extraLarge: 32,
+  huge: 48,
+  massive: 64,
+};
```

Comment:
We plan to remove spacing from the theme in a different PR, but for the moment, I just got it out of the way.